### ISSUE TITLE
Volts/millivolts inconsistencies

### ISSUE THREAD

**Issue by thomas-phillips-nz** [kxtells/tenma-serial#14]
Hi

I noticed that the code is quite inconsistent with volts/millivolts. All the functions expect millivolts/milliamps, but the read/running functions return values in volts/amps.

I don't want to just submit a PR to change these as it might break existing things that use this library, but should we consider changing to be consistently millivolts/milliamps when setting and returning? Or should we be consistently volts/amps when setting/returning?

Just looking to start the discussion.

**Comment by kxtells** [comment]
You are right, write functions work with `millis` while read functions simply return whatever the unit returned (`volts` or `amps` in this case). Luckily we don't have write functions that expect both `millis` and `amps`, that would be extra confusing :-D

The original version is already 6yo and right now I don't recall if there was any specific reason for this decision.

Any of those changes risk breaking existing scripts running somewhere, but we could even jump the version to `2.0` signaling that there's actually an API change.

If that was the case case, I think the easiest approach is to just change the return to `millis` (so we're consistent in read an write units). And then, if we want to be fancier, we can always add a configuration option on class instantiation setting the expected `input/output` units, but I think that's an overkill.

**Comment by kxtells** [comment]
I mark this one for the things to take into account if we start a 2.x version with cleaned up API.
